webapp/src/main/java/com/box/l10n/mojito/entity/ThirdPartyFileChecksum.java

```diff
@@ -0,0 +1,69 @@
+package com.box.l10n.mojito.entity;
+
+import javax.persistence.Column;
+import javax.persistence.Embedded;
+import javax.persistence.Entity;
+import javax.persistence.Index;
+import javax.persistence.PreUpdate;
+import javax.persistence.Table;
+import org.joda.time.DateTime;
+
+/** Entity that stores the checksum of a translated file downloaded via a third party sync. */
+@Entity
+@Table(
+    name = "third_party_sync_file_checksum",
+    indexes = {
+      @Index(
+          name = "I__THIRD_PARTY__CHECKSUM__REPOSITORY_ID__FILE_NAME__LOCALE_ID",
+          columnList = "repository_id, locale_id, file_name",
+          unique = true),
+    })
+public class ThirdPartyFileChecksum extends AuditableEntity {
+
+  @Embedded private ThirdPartyFileChecksumCompositeId thirdPartyFileChecksumCompositeId;
+
+  @Column(name = "md5")
+  private String md5;
+
+  public ThirdPartyFileChecksum() {}
+
+  public ThirdPartyFileChecksum(
+      ThirdPartyFileChecksumCompositeId thirdPartyFileChecksumCompositeId, String md5) {
+    this.thirdPartyFileChecksumCompositeId = thirdPartyFileChecksumCompositeId;
+    this.md5 = md5;
+  }
+
+  @PreUpdate
```

Comment:
Do we need this? I don't see it used in current code? maybe it was need before extending AuditableEntity?

Reply:
Not anymore, I was originally using `SettableAuditableEntity` which doesn't handle `last_modified_date` so needed it then but will remove it now.